packages/react-storage/src/components/StorageBrowser/controls/types.ts

```diff
@@ -20,5 +20,10 @@ export interface ControlsContext {
   actionsConfig: {
     type: 'SINGLE_ACTION' | 'BATCH_ACTION';
     isCancelable: boolean;
+    actionStart?: {
+      onClick?: () => void;
+      label?: string;
+      disabled?: boolean;
+    };
```

Comment:
@calebpollman could you verify this is at least sort of the right place for this? I know this is all still in flux so just want to know if this is amenable as a solution now or if it's way off base and we could do better even in the interim